ess-controller/start.sh

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env bash
+
+source /home/saluser//miniforge/bin/activate
```

Comment:
Is that double slash supposed to be a single one?

Reply:
It is. Thanks for catching that!